explicit_single_phase/validation/sliding_block_inclined_boundary/data/temp.txt

```diff
@@ -0,0 +1,2 @@
+temp file
+TODO : remove me
```

Comment:
What is this used for?

Reply:
wanted the `data` directory to show up so just putting something small in there until we actually write the README. It will be used to store images/gifs for the README (e.g. stress contour plot, convergence graph, etc)

Reply:
OK!